unity/README.md

```diff
@@ -0,0 +1,17 @@
+# .NET Runtime - Unity Details
+
+This is Unity's fork of the .NET Runtime repository.
+
+The difference between this fork and the upstream repository should be as small as possible, with all of the differences specific to Unity. Our goal is to upstream as many changes made here as possible.
+
+## Pulling changes from upstream
+
+There is a job in Unity's internal CI which runs weekly to pull the latest code from the upstream [dotnet/runtime](https://github.com/dotnet/runtime) repository `main` branch and create a pull request to merge these changes to the [`unity-main`](https://github.com/Unity-Technologies/runtime/tree/unity-main) branch.
```

Comment:
Should we add a link to the Yamato job? (We have a job, just not telling you where it is.  :) )

Reply:
I was thinking about that as well. I did not add it because the Yamato jobs are not accessible publicly.

Reply:
I did forget this was public.